plugin/storage/grpc/config/config.go

```diff
@@ -78,6 +79,9 @@ func (c *Configuration) Close() error {
 		c.pluginHealthCheck.Stop()
 		c.pluginHealthCheckDone <- true
 	}
+	if c.remoteRPCClient != nil {
```

Comment:
To pass the code coverage, I set up a gRPC server within the unit test for the factory to use, since we can't just mock the gRPC configuration. Because the current gRPC storage doesn't close the remote gRPC client, I added a method to close the client right inside the factory's `Close()` function. However, it looks like the code coverage tool isn't recognizing this particular line as covered. Even though I've called the Close() function in my unit test, which should hit this line, the method is abstracted behind an interface. Could this be why it's not showing up as covered?

Reply:
in Go coverage is computed in a weird way, it's package local, i.e. coverage is only collected for a package from tests running in the same package. This behavior can be altered by passing additional packages explicitly, as we do for storage integration tests (we tell Go to count coverage in all other packages):
```
$(GOTEST) -coverpkg=./... -coverprofile cover.out $(STORAGE_PKGS)
```
So when the integration test runs for grpc-plugin it should include your Close changes, but not if you just run `go test` manually without `-coverpkg`